src/panels/lovelace/cards/hui-entities-card.ts

```diff
@@ -72,7 +74,7 @@ class HuiEntitiesCard extends hassLocalizeLitMixin(LitElement)
       }
     }
 
-    this._config = config;
+    this._config = { theme: "default", ...config };
```

Comment:
This is bad.

It means that if I specify a theme on my view, it will be locally overridden by the default theme in this card.

Reply:
Hmm this is how we are doing it in glance and button so that could be a problem. But I theme my overall Lovelace and I'm using both those cards and the theme sticks

Reply:
But I don't specify it on a view. Will test later

Reply:
This seems to work as expected. I put a theme on the view and all cards show that theme. I set a theme on the entities card and the theme changes only for that card.

Reply:
Looked at the implementation, it's actually pretty brilliant. You're right, it is fine.